testhelpers/testhelpers.go

```diff
@@ -136,11 +144,16 @@ func Eventually(t *testing.T, test func() bool, every time.Duration, timeout tim
 	}
 }
 
-func PullImage(dockerCli dockercli.CommonAPIClient, ref string) error {
-	rc, err := dockerCli.ImagePull(context.Background(), ref, dockertypes.ImagePullOptions{})
+func PullImage(dockerCli dockercli.CommonAPIClient, ref string, registryAuth ...string) error {
```

Comment:
It doesn't look like any of the tests are actually passing a registryAuth

Reply:
Fixed!